pom.xml

```diff
@@ -117,6 +121,72 @@
           <projectId>GCLOUD_CONFIG</projectId>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>com.github.eirslett</groupId>
+        <artifactId>frontend-maven-plugin</artifactId>
+        <version>1.9.1</version>
+        <executions>
+          <execution>
+            <id>Install node and npm</id>
+            <goals>
+              <goal>install-node-and-npm</goal>
+            </goals>
+            <phase>generate-resources</phase>
+            <configuration>
+              <nodeVersion>v10.14.2</nodeVersion>
+              <npmVersion>6.14.8</npmVersion>
+            </configuration>
+          </execution>
```

Comment:
nit: add a blank line between each logical block, for readability purposes

Reply:
Blank lines added between <code>execution</code> blocks with commit c980571